docs/sample_config.yaml

```diff
@@ -1075,6 +1075,25 @@ url_preview_accept_language:
 #   - en
 
 
+oembed:
+    # By default, the providers.json from https://oembed.com/ is included
+    # with Synapse.
```

Comment:
I wasn't quite sure what to put here, I didn't want to say it is *just* Twitter since hopefully we'll update it in the future? 🤷 I made it a bit more generic though.